test/automated_test.py

```diff
@@ -91,6 +92,30 @@ def check_values(list1, list2, atol=atol, rtol=rtol):
         assert torch.allclose(g1, g2, atol=atol, rtol=rtol)
 
 
+def make_simple_cnn_setting(act_cls, bias):
+    """
+    input: activation function & bias setting
+    return: CNN network with torch.nn.Conv2d and torch.nn.Linear
+    """
+
+    def make_simple_cnn(act_cls, bias):
+        return torch.nn.Sequential(
+            torch.nn.Conv2d(3, 2, 2, bias=bias),
+            act_cls(),
+            torch.nn.Flatten(),
+            torch.nn.Linear(72, 5),
+        )
+
+    dict_setting = {
+        "input_fn": lambda: torch.rand(3, 3, 7, 7),
+        "module_fn": lambda: make_simple_cnn(act_cls, bias),
+        "loss_function_fn": lambda: torch.nn.CrossEntropyLoss(),
+        "target_fn": lambda: classification_targets((3,), 5),
+    }
```

Comment:
It would be good to label automatically generated settings as such. We can do that with the `'id_prefix="automated-simple-cnn"'`.